CHANGELOG.rst

```diff
@@ -2,6 +2,195 @@
 Release Notes
 =============
 
+.. _Release Notes_7.31.0:
+
+7.31.0 / 6.31.0
+======
+
+.. _Release Notes_7.31.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2021-09-13
+
+- Please refer to the `7.31.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7310>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.31.0_New Features:
+
+New Features
+------------
+
+- Added `hostname_file` as a configuration option that can be used to set
```

Comment:
I believe single backticks do not have the intended effect in the `rst` format (they are [interpreted text](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#interpreted-text) and not [inline litterals](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-literals)). Should they be changed to double backticks?

Reply:
If so - those changes are made automatically by script, so it may be good to double check after the release whether there is something we can do about it.